src/lib/polyfill/import.js

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
Mostly curious—I was wondering why you chose not to use the [dynamic-import-polyfill](https://github.com/GoogleChromeLabs/dynamic-import-polyfill)?

Reply:
No polyfill is actually "correct", because they cannot resolve relative paths. They also do more work, because they care about resolving the results of an import, whereas we explicitly _do not_ need that.

It's up to you, but in general I think a few well-commented lines here are clearer than a magic black box that is basically doing the same thing.

Reply:
yep that's cool with me 👍